descartes_planner/src/planning_graph.cpp

```diff
@@ -41,7 +41,7 @@ using namespace descartes_trajectory;
 namespace descartes_planner
 {
 
-const double MAX_JOINT_DIFF = M_PI;
+const double MAX_JOINT_DIFF = M_PI / 20;
```

Comment:
Hard coding this limit scares me.  Physically, it should be a joint velocity limit defined by the robot.  Thus it should be dependent on time and the configuration of the robot.  Is there anyway to expose this at a higher level?